websites/P/Pony Town/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Lavenderleaff",
+		"id": "205984221859151873"
+	},
+	"contributors": [
+		{
+			"name": "joerkig",
+			"id": "205984221859151873"
+		}
+	],
```

Comment:
A contributor must have contributed to at least 20% of the code base. This is unlikely because it is a new PR